lib/index.js

```diff
@@ -23,7 +24,8 @@ module.exports = function (startPath, ignore) {
 		} catch (err) {}
 
 		nextPath = path.join(searchPath + '/..');
-		if (nextPath === path.normalize('/') || nextPath === '.' || nextPath === '..') {
+		const root = (os.platform() === 'win32') ? nextPath.split(path.sep)[0] + path.sep : path.normalize('/');
```

Comment:
Thanks a lot for the pull request! Just one tiny nit: Could you perhaps add a little comment here that clarifies this line? Something like "if Windows, uses main drive's identifier as root path"? Thanks!

Reply:
> Thanks a lot for the pull request! Just one tiny nit: Could you perhaps add a little comment here that clarifies this line? Something like "if Windows, uses main drive's identifier as root path"? Thanks!

Sure, something like this?
https://github.com/bertyhell/parent-package-json/blob/a69451531fba2ed578e54336ba39c906296ab9e7/lib/index.js#L27-L28

Reply:
Yup, looks good to me, thanks! :+1: